src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1357,6 +1414,24 @@ def validate(self) -> None:
         if self.transports not in ['tcp']:
             raise SpecValidationError('Invalid transport. Valid values are tcp')
 
+        if self.log_level:
+            if self.log_level not in ['debug', 'DEBUG',
+                                      'info', 'INFO',
+                                      'warning', 'WARNING',
+                                      'error', 'ERROR',
+                                      'critical', 'CRITICAL']:
+                raise SpecValidationError(
+                    'Invalid log level. Valid values are: debug, info, warning, error, critial')
```

Comment:
NIT, but for consistency's sake, it would be nice to capitalize debug, info, etc. as is done in the other error message.